packages/sil-governance/src/routes/create-proposal/components/ProposalForm.tsx

```diff
@@ -111,6 +114,29 @@ const ProposalForm = (): JSX.Element => {
           quorum,
         };
       }
+      case ProposalType.AddValidator: {
+        const pubkeyArray = values[PUBKEY_ADD_FIELD].split("_");
+        const algo = pubkeyArray[0] === Algorithm.Ed25519 ? Algorithm.Ed25519 : Algorithm.Secp256k1;
```

Comment:
At IOV, all validator keypairs will be `Algorithm.Ed25519`, so you can hardcode the algorithm. The user then just types in a 32 bytes hex pubkey like e.g. `2fe961606ebda309259dc116bbb7d34d573b5617b203f578c4d221c946992c5e`